app/views/matches/index.html.slim

```diff
@@ -7,8 +7,10 @@
   table.match-tbl
     thead
       tr
-        th[colspan="5"]
-          = match['date']
+        - if @date_var != match['date']
+          - @date_var = match['date']
```

Comment:
hmm is this condition necessary? how about just do `@date_var = match['date']` ?